symplyphysics/laws/thermodynamics/work_done_by_gas_in_isobaric_process.py

```diff
@@ -0,0 +1,34 @@
+from sympy import (Eq, solve)
```

Comment:
`isobaric process` in file name may be hard to decipher. Why not use `constant pressure` instead?